src/routes/NonAuthLayout/OAuth/Consent/Consent.scss

```diff
@@ -0,0 +1,129 @@
+.topRow {
+  position: absolute;
+  top: 0px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 20px;
+}
+
+.container {
+  background-color: white;
+  box-shadow: 0px 0px 6px 0px $color-black-10;
+  padding: 50px 40px 40px;
+  border-radius: 8px;
+  width: 420px;
+}
+
+.formWrapper {
+  padding: 30px;
+}
+
+.title {
+  composes: hdr-headline from 'css/typography.scss';
+  color: $color-caribbean-green;
+  text-align: center;
+  margin-bottom: 40px;
+}
+
+.field {
+  display: block;
+  margin-bottom: 20px;
+  position: relative;
+
+  .field-label {
+    color: rgba(128, 140, 156, 1.00);
+    font-size: 16px;
+    display: block;
+    position: absolute;
+    top: 12px;
+    left: 15px;
+    transition: all .25s ease;
+  }
+
+  .authInput {
+    height: 50px;
+
+    input {
+      height: 50px;
+      border: 1px solid rgba(225, 229, 233, 1.00);
+      background-color: rgba(251, 252, 253, 1.00);
+      padding-top: 8px;
+    }
+
+    input:-webkit-autofill, input:-internal-autofill-selected {
+      animation-name: onAutoFillStart;
+    }
+
+    input:not(:-webkit-autofill) {
+      animation-name: onAutoFillCancel;
+    }
+
+  }
+
+  .field-label.active, .field-label:-webkit-autofill {
+    font-size: 11px;
+    top: 4px;
+    left: 16px;
+  }
+}
+
+/* the only way to detect autocomplete */
```

Comment:
Good! Wish every site seemed to catch-on. It's incredible how are still dumb to autocomplete...